week-8/Homework/mandatory/4-programmer-humour/script.js

```diff
@@ -0,0 +1,22 @@
+fetch('https://xkcd.now.sh/?comic=latest')
+    .then(function(response) {
+        return response.json();
+    })
+    .then(function(obj) {
+        console.log(obj);
+
+
+        addImgToDOM(obj.img);
+    })
+    .catch((error) => {
+        console.log(error);
+    })
+
+const addImgToDOM = (image) => {
+    const imgElement = document.createElement('img');
+    imgElement.src = image;
+    imgElement.style.display = "block";
+    imgElement.style.marginRight = "auto";
+    imgElement.style.marginLeft = "auto";
```

Comment:
In this simple page, you can attach a general CSS tag role to the img element.
Eg.
```
img {
 display: block;
 margin-right: auto;
 margin-left: auto;
}
```